src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -192,4 +241,65 @@ describe('handle-key-press', () => {
       });
     });
   });
+
+  describe('headHandleKeyPress', () => {
+    const sandbox = sinon.createSandbox();
+    let rowIndex;
+    let colIndex;
+    let evt = {};
+    let rootElement = {};
+    let changeSortOrder;
+    let layout;
+    let isDim;
+
+    beforeEach(() => {
+      rowIndex = 0;
+      colIndex = 0;
+      evt = {
+        key: 'ArrowDown',
+        stopPropagation: sinon.spy(),
+        preventDefault: sinon.spy(),
+        target: {
+          blur: sinon.spy(),
+          setAttribute: sinon.spy(),
+        },
+      };
+      rootElement = {
+        getElementsByClassName: () => [{ getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] }],
+      };
+      changeSortOrder = sinon.spy();
+      sandbox.replace(handleKeyPressObject, 'moveFocus', sandbox.spy());
+    });
+
+    afterEach(() => {
+      sandbox.verifyAndRestore();
+    });
+
+    describe('ArrowDown', () => {
+      it('should prevent default behavior, remove current focus and set focus and attribute to the next cell', () => {
+        headHandleKeyPress(evt, rootElement, rowIndex, colIndex);
+        expect(handleKeyPressObject.moveFocus).to.have.been.calledOnce;
```

Comment:
@haxxmaxx  I can not figure out why I got ' AssertionError: expected spy to have been called exactly once, but it was called 0 times' here. Could I have your help to have a look?